### ISSUE TITLE
Incorrect User Home Folder

### ISSUE THREAD

**Issue by webadict** [demilich1/metastone#175]
I got an odd error while trying to open MetaStone on a new computer. I'm using Windows 8.1 if it matters. I'm using the newest release, as well.

```
10:58:07 ERROR - Trouble creating C:\Users\mladebeck\Documents\metastone\cards
10:58:07 ERROR - Trouble creating C:\Users\mladebeck\Documents\metastone\decks
```

I'm not entirely sure why it's not using MY user home folder, especially considering it usually works. Anyone else get this issue with a new installation?

**Comment by webadict** [comment]
Other notes: I'm using the JAR file for the v1.1.0RC3 release. I'm doing some searching to see if anything else might be up.

**Comment by webadict** [comment]
Taking a guess here... Not sure how right I am.

#164 updated MetaStone to generate a BuildConfig class on run. However, the current JAR file has a BuildConfig file packaged within it. My guess is that it won't generate a BuildConfig class file since one exists, meaning that my user directory won't be reset and uses the one already there.

My guess is that the JAR file needs the BuildConfig class removed. I'll test it out.

**Comment by webadict** [comment]
Hmm... No, looks like I'm not correct there. I deleted the BuildConfig class file and it still gives the same error. Bizarre.

**Comment by akoscz** [comment]
awwshit... BuildConfig is a build time generated file, in other words it gets created when someone builds the app and in turn it uses their Home dir.    This is fine if people are building their own version of Metastone from source.  This will NOT work if we distribute a stand alone binary for people to use.   

I will think about a solution and code something up later tonight.

Please label this is as a bug.

**Comment by demilich1** [comment]
This should be fixed by #176
